org.osgi.service.servlet/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -1,28 +1,71 @@
 eclipse.preferences.version=1
+enableParallelJavaIndexSearch=true
```

Comment:
We need to keep all the project's .settings in sync. So we should not commit this.